integration/teardown/teardown.go

```diff
@@ -15,7 +15,8 @@ func Teardown(c *client.AzureClientSet, g *framework.Guest, h *framework.Host) e
 	var err error
 
 	{
-		h.DeleteGuestCluster("azure-operator", "azureconfig", "removing finalizer from runtime object")
+		// TODO the deletion detection should rather happen based on the cluster status or even based on the CR being gone
```

Comment:
If this is a long term TODO should there be an issue? If its short term I'd still like to see a sentence with a `.`.

Reply:
Good catch. I was lazy here. The issue is https://github.com/giantswarm/giantswarm/issues/3839. Added it to the comment in the code.